Loop/Managers/DeviceDataManager.swift

```diff
@@ -1313,10 +1313,11 @@ extension DeviceDataManager {
     func handleRemoteNotification(_ notification: [String: AnyObject]) {
 
         defer {
```

Comment:
Prefixing remote logs in this method to make them easy to search in logs.

Someday I'd like to refactor this method to throw an error (or return a Result<>) so that logging can happen in a second statement.